runner/src/main.rs

```diff
@@ -33,9 +33,9 @@ use internal::*;
 fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
If it possible to run a server or examples using the `runner`, or it only runs Cargo related tasks and Bazel builds?

Reply:
I am going to add that functionality in the next PR.